pkgs/development/interpreters/erlang/generic-builder.nix

```diff
@@ -81,7 +82,7 @@ in stdenv.mkDerivation ({
     ./otp_build autoconf
   '';
 
-  configureFlags = [ "--with-ssl=${openssl.dev}" ]
+  configureFlags = [ "--with-ssl=${opensslPackage.dev}" ]
```

Comment:
We have helper function to fallback to the default output:

```suggestion
  configureFlags = [ "--with-ssl=${lib.getDev opensslPackage}" ]
```

Reply:
Thanks a lot!